portal-kernel/src/com/liferay/portal/kernel/layoutconfiguration/util/RuntimePage.java

```diff
@@ -30,7 +30,7 @@
 @ProviderType
 public interface RuntimePage {
 
-	public LayoutTemplate getLayoutTemplate(String velocityTemplateId);
+	public LayoutTemplate getLayoutTemplate(String templateId);
```

Comment:
We shall change this in [RuntimePageUtil](https://github.com/liferay/liferay-portal/blob/0a9951c25d434cc515f620ffba68c3a9e4fbaeed/portal-kernel/src/com/liferay/portal/kernel/layoutconfiguration/util/RuntimePageUtil.java#L30) as well